packages/suite/src/views/wallet/sign-verify/index.tsx

```diff
@@ -74,155 +73,187 @@ interface FormEvent {
-                                type="text"
-                                state={verifyAddressError ? 'error' : undefined}
-                                bottomText={verifyAddressError ? verifyAddressError.message : null}
-                            />
-                        </Row>
-                        <Row>
-                            <Textarea
-                                topLabel={intl.formatMessage(messages.TR_MESSAGE)}
-                                name="verifyMessage"
-                                value={verifyMessage}
-                                onChange={this.handleInputChange}
-                                rows={4}
-                                maxRows={4}
-                                maxLength={255}
-                            />
-                        </Row>
-                        <Row>
-                            <Textarea
-                                topLabel={intl.formatMessage(messages.TR_SIGNATURE)}
-                                name="verifySignature"
-                                value={verifySignature}
-                                onChange={this.handleInputChange}
-                                rows={4}
-                                maxRows={4}
-                                maxLength={255}
-                            />
-                        </Row>
-                        <RowButtons>
-                            <StyledButton onClick={signVerifyActions.clearVerify}>
-                                <Translation id="TR_CLEAR" />
-                            </StyledButton>
-                            <StyledButton
-                                // isDisabled={!!verifyAddressError || !device.connected}
-                                isDisabled
-                                onClick={() => {
-                                    if (errors.length <= 0) {
-                                        signVerifyActions.verify(
-                                            verifyAddress,
-                                            verifyMessage,
-                                            verifySignature,
-                                        );
-                                    }
-                                }}
-                            >
-                                <Translation id="TR_VERIFY" />
-                            </StyledButton>
-                        </RowButtons>
-                    </Verify>
-                </Wrapper>
-            </WalletLayout>
-        );
-    }
-}
+    const handleInputChange = (event: FormEvent) => {
+        signVerifyActions.inputChange(event.target.name as InputNameType[0], event.target.value);
+    };
+
+    const getAddress = () => {
+        if (
+            selectedAccount.account.networkType === 'bitcoin' &&
+            typeof selectedAccount.account.addresses?.unused[0]?.address === 'string' &&
+            typeof selectedAccount.account.addresses?.unused[0]?.address === 'string'
```

Comment:
why `typeof selectedAccount.account.addresses?.unused[0]?.address === 'string'` twice?

Reply:
it is mistake, path must be on this place